source-browser.html

```diff
@@ -181,7 +209,46 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(topic) {
+        $('.li').append(
+          $('<a>', {
+            id: `li${topic}`,
+            href: `#${topic}`
+          }).text(topic)
+        );
+      }
+
+      let list = [];
+      $(document).ready(() => {
+	    TOPICS.forEach(item => {
+	      let heading = item.name;
+	      list.push(heading);
```

Comment:
This `list` seems completely unnecessary. `TOPICS` can be used directly below instead.

Reply:
What method for TOPICS should I use?  TOPICS. push(heading) doesn't work.

Reply:
What method for TOPICS should I use?  TOPICS. push(heading) doesn't work.

Reply:
`TOPICS.forEach` can be used directly below.

Reply:
Sorry, I meant 

TOPICS.name.forEach(topic => {

doesn't work.

Reply:
It can work if you use topic.name later.

Reply:
That makes sense.   When I do TOPICS.forEach, though, it just gives [object Object] as the footer item names.

Reply:
![screen shot 2018-11-17 at 12 40 07 pm](https://user-images.githubusercontent.com/44486178/48665579-f6eb1c80-ea65-11e8-923b-95ea6fdb5f67.png)

Reply:
Try using the debugger.

Reply:
Okay, I managed to get some of it working—the titles are there now, but the arrows are gone.

Reply:
Debugger :)

Reply:
It works now!